packages/xrpl/src/models/ledger/PermissionedDomain.ts

```diff
@@ -0,0 +1,19 @@
+import { AuthorizeCredential } from '../common'
+
+import { BaseLedgerEntry, HasPreviousTxnID } from './BaseLedgerEntry'
+
+export default interface PermissionedDomain
+  extends BaseLedgerEntry,
+    HasPreviousTxnID {
+  LedgerEntryType: 'PermissionedDomain'
+
+  Owner: string
+
+  Flags: 0
+
+  OwnerNode: string
+
+  Sequence: number
+
+  AcceptedCredentials: AuthorizeCredential[]
```

Comment:
Easier to read to put transaction specific fields at top and matches the rest of the models in the codebase.

```suggestion
  LedgerEntryType: 'PermissionedDomain'

  Owner: string

  AcceptedCredentials: AuthorizeCredential[]

  Flags: 0

  OwnerNode: string

  Sequence: number
```

Reply:
[2080c21](https://github.com/XRPLF/xrpl.js/pull/2874/commits/2080c21bd425a596f844d2898ed65c465f19adfd)